src/App.jsx

```diff
@@ -12,4 +13,4 @@ const App = () => (
   </div>
 );
 
-export default App;
+export default connect()(App);
```

Comment:
@Gwillison415 This is over my head, can you tell me why the need here to connect the app component?

Reply:
``` mapStateToProps``` and``` mapDispatchToProps ``` are both pure functions that are provided the stores “state” and “dispatch” respectively.  only components within the Provider can be connected however our app.jsx is yet to change the state of the application. When we institute a login componentWillMount inside of ```Login.jsx``` the App will conditionally render based on login and need the connect function that takes mpstp and mdtp. Does this help clarify? I'm not sure if this was for testing but for linting.. 5 days ago.. what say you @HamidAghdaee ?

Reply:
indeed @Gwillison415 why connect here?